program/Cargo.toml

```diff
@@ -1,21 +1,23 @@
 [package]
 name = "paladin-lockup-program"
-version = "0.1.0"
+version = "0.0.1"
```

Comment:
is there any reason to bump the version down to 0.0.1? I think this is at least 0.1.0, and we can 1.0.0 after audits

Reply:
I've been making everything 0.0.1 until we or Paladin were ready to publish. The clients are all 0.0.1.